### ISSUE TITLE
Quick Reblog: already-reblogged tick missing

### ISSUE THREAD

**Issue by AprilSylph** [AprilSylph/XKit-Rewritten#750]
When the new reblog controls test is enabled, the reblog button gets turned the appropriate colour, but the tick for accessibility is missing.

This may be what was being referenced in https://github.com/AprilSylph/XKit-Rewritten/issues/742#issuecomment-1236721360.
